syntax = "proto3";

package dispatch;

message DispatchMessage {
    string module_id = 1;
    string function_id = 2;
    bytes payload = 3;
}